core/src/main/scala/com/databricks/labs/remorph/generators/sql/DataTypeGenerator.scala

```diff
@@ -33,7 +33,7 @@ object DataTypeGenerator {
     case ir.ArrayType(elementType) => sql"ARRAY<${generateDataType(ctx, elementType)}>"
     case ir.StructType(fields) =>
       val fieldTypes = fields
-        .map { case ir.StructField(name, dataType, nullable) =>
+        .map { case ir.StructField(name, dataType, nullable, metadata) =>
```

Comment:
```suggestion
        .map { case ir.StructField(name, dataType, nullable, _) =>
```
use `_` for vars you don't use

Reply:
done